recipe/meta.yaml

```diff
@@ -31,9 +31,9 @@ requirements:
     - numpy
     - python >={{ python_min }}
     - requests
-    - scikit-image
+    - scikit-image >=0.19.0
     - toolz
-    - zarr >=2.8.1,<3
+    - zarr >=v3.0.0
```

Comment:
The bot (via the setting `inspection: update-grayskull`) correctly copied the requirement `zarr >=v3.0.0` from the upstream [`pyproject.toml`](https://github.com/ome/ome-zarr-py/blob/v0.12.0/pyproject.toml#L22) for ome-zarr 0.12.0. Unfortunately while `pip` understands that `v3.0.0==3.0.0`, conda does not, which effectively removes this required lower bound, leading to broken environments.

Fixed in https://github.com/conda-forge/ome-zarr-feedstock/pull/31 and https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/1068

Reply:
Also hoping to fix this upstream via https://github.com/ome/ome-zarr-py/pull/487, since the bot is going to add back the `v` to the version string on every update PR